app/controllers/api/v2/courses_controller.rb

```diff
@@ -138,7 +138,7 @@ def has_synced?
           SearchAndCompareAPIService::Request.sync([@course])
         else
           raise RuntimeError.new(
-            'course is not syncable'
+            "course '#{@course}' is not syncable due to: #{@course.errors.details}"
```

Comment:
`"#{@course.name} (#{@course.code}) sync error: #{@course.errors.details}"` ?

Reply:
How about with provider
```ruby
"'#{@course}' '#{@course.provider}' sync error: #{@course.errors.details}" 
```

it gives
```ruby
# it gives
course_name (course_code) provider_name (provider_code) ....
'G (C3D3)' 'ACME SCITT28921 (A1)' sync error: {:dfe_subjects=>[{:error=>"No DfE subject."}]}
```